### ISSUE TITLE
Getting very less accuracy for Image 

### ISSUE THREAD

**Issue by kumarsiddappa-git** [mrdbourke/pytorch-deep-learning#753]
'''  
class FashionMNIStModelV0(nn.Module):
  def __init__(self,input_features,hidden_units,output_features):
    super().__init__()
    self.layer_stack=nn.Sequential(
        nn.Flatten(),
        nn.Linear(in_features=input_features,out_features=hidden_units),
        nn.Linear(in_features=hidden_units,out_features=output_features)
    )

  def forward(self,x):
    return self.layer_stack(x)
'''

model_0=FashionMNIStModelV0(input_features=784,hidden_units=10,output_features=len(class_names)).to("cpu")

My Training Model

'''
from tqdm.auto import tqdm

torch.manual_seed(42)
train_time_start_on_cpu = timer()

#set the number of pochs (we'll keep this small for faster raining time)
epochs=3

#create trining and test loop
for epoch in range(epochs):
  print(f"Epoch:{epoch}\n------")

  #trianing
  train_loss = 0

  #Add a loop tp loop through the training batches
  for batch,(X,y) in enumerate(train_dataloader):
    model_0.train()

    #Forward pass
    y_pred = model_0(X)

    loss = loss_fn(y_pred,y)

    train_loss+=loss

    optimizer.zero_grad()

    loss.backward()

    optimizer.step()

    #Print the data
    if batch %400 ==0 :
      print(f"Looked at {batch *len(X)}/{len(train_dataloader.dataset)} samples.")

    #Divide the train loss
  train_loss /= len(train_dataloader)
    #testing 
  test_loss,test_acc=0,0
  model_0.eval()

  with torch.inference_mode():
    for X_test,y_test in test_dataloader:
      y_test_pred = model_0(X_test)

      test_loss = loss_fn(y_test_pred,y_test)

      test_loss+=test_loss

      test_acc= accuracy_fn(y_true=y_test,y_pred=y_test_pred.argmax(dim=1))

    #calaulte the test loss average per batch
    test_loss/=len(test_dataloader)

    test_acc/=len(test_dataloader)

  #Print out what happens
  print(f"\n Train loss:{train_loss:.4f} | Test Loss:{test_loss:.4f} | Test acc :{test_acc:.4f}")

  #calulate trining time 
  train_time_end_on_cpu = timer()
  total_train_time_model_0 = print_train_time(start=train_time_start_on_cpu,
                                              end=train_time_end_on_cpu,
                                              device=str(next(model_0.parameters()).device)
  )
  print(f"Total time : {total_train_time_model_0}")

'''

Output 

"""

Epoch:0
------
Looked at 0/60000 samples.
Looked at 12800/60000 samples.
Looked at 25600/60000 samples.
Looked at 38400/60000 samples.
Looked at 51200/60000 samples.

 Train loss:0.4307 | Test Loss:0.0042 | Test acc :0.2596
Train time on cpu: 13.978 seconds
Total time : 13.977587818999382
Epoch:1
------
Looked at 0/60000 samples.
Looked at 12800/60000 samples.
Looked at 25600/60000 samples.
Looked at 38400/60000 samples.
Looked at 51200/60000 samples.

 Train loss:0.4246 | Test Loss:0.0041 | Test acc :0.2596
Train time on cpu: 26.074 seconds
Total time : 26.073535761999665
Epoch:2
------
Looked at 0/60000 samples.
Looked at 12800/60000 samples.
Looked at 25600/60000 samples.
Looked at 38400/60000 samples.
Looked at 51200/60000 samples.

 Train loss:0.4218 | Test Loss:0.0042 | Test acc :0.2396
Train time on cpu: 38.908 seconds
Total time : 38.908417628999814

"""

Can some check and let me know

**Comment by kumarsiddappa-git** [comment]
```
with torch.inference_mode():
    for batch,(X_test,y_test) in enumerate(test_dataloader):
      y_test_pred = model_0(X_test)

      test_loss = loss_fn(y_test_pred,y_test)
                  
      test_loss+=test_loss

      test_acc= accuracy_fn(y_true=y_test,y_pred=y_test_pred.argmax(dim=1))
    
      if batch% 10 ==0:
        print(f"batch : {batch} | test_acc :{test_acc:.3f}")


```


getting a good accuracy 

batch : 0 | test_acc :81.250
batch : 10 | test_acc :81.250
batch : 20 | test_acc :81.250
batch : 30 | test_acc :87.500
batch : 40 | test_acc :93.750
batch : 50 | test_acc :93.750
batch : 60 | test_acc :81.250
batch : 70 | test_acc :81.250
batch : 80 | test_acc :75.000
batch : 90 | test_acc :78.125
batch : 100 | test_acc :68.750
batch : 110 | test_acc :84.375
batch : 120 | test_acc :87.500
batch : 130 | test_acc :75.000
batch : 140 | test_acc :65.625
batch : 150 | test_acc :75.000
batch : 160 | test_acc :84.375
batch : 170 | test_acc :75.000
batch : 180 | test_acc :81.250
batch : 190 | test_acc :84.375
batch : 200 | test_acc :90.625
batch : 210 | test_acc :84.375
batch : 220 | test_acc :90.625
batch : 230 | test_acc :87.500
batch : 240 | test_acc :93.750
batch : 250 | test_acc :87.500
batch : 260 | test_acc :78.125
batch : 270 | test_acc :81.250
batch : 280 | test_acc :75.000
batch : 290 | test_acc :78.125
batch : 300 | test_acc :87.500
batch : 310 | test_acc :90.625

**Comment by kumarsiddappa-git** [comment]
my data loader 

```
## BATCHSIZE
from torch.utils.data import DataLoader
BATCHSIZE=32

train_dataloader = DataLoader(dataset=train_data,
                              batch_size=BATCHSIZE,
                              shuffle=True)

test_dataloader = DataLoader(dataset=test_data,
                             batch_size=BATCHSIZE,
                             shuffle=True)

train_dataloader , test_dataloader

```

**Comment by cheng128** [comment]
Hi @kumarsiddappa-git, I had the same problem and modified the learning rate to a smaller number like `0.005`. The accuracy looked better than `lr=0.1`. I hope this also works for you.
